libs/checkout/src/index.ts

```diff
@@ -46,6 +47,7 @@ export { DaffBillingReducersState } from './billing/reducers/billing-reducers.in
 export { DaffBillingReducerState } from './billing/reducers/billing/billing-reducer.interface';
 export { daffBillingReducers } from './billing/reducers/billing-reducers';
 export { daffBillingReducer } from './billing/reducers/billing/billing.reducer';
+export { BillingContainer } from './billing/containers/billing.component';
```

Comment:
These (and other changes like it) caused errors where angular didn't like that some things were used in exported modules, but were not exported to the end user.